<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 199, 199);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
background-color: rgb(255, 255, 127);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="Star_Total">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
background-color: rgb(232, 232, 232);

font: 10pt &quot;Consolas&quot;;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Review_Count_total">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
background-color:  rgb(232, 232, 232);


font: 10pt &quot;Consolas&quot;;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>340</y>
     <width>841</width>
     <height>481</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: rgb(232, 232, 232);
border-radius: 5px;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;

</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>별점 트렌드</string>
    </attribute>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;

font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;

border-radius: 15px;
background-color:  rgb(134, 199, 99);</string>
     </property>
     <property name="text">
      <string>사이트 평점 Vs. 학습한 평점 비교</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>761</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>별점/댓글 일치율</string>
    </attribute>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;color: black;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;font: 75 9pt &amp;quot;나눔고딕 ExtraBold&amp;quot;;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;

font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;

border-radius: 15px;
background-color:  rgb(134, 199, 99);</string>
     </property>
     <property name="text">
      <string>불일치 댓글 예시</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>381</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;color: black;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;font: 75 9pt &amp;quot;나눔고딕 ExtraBold&amp;quot;;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;

font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;

border-radius: 15px;
background-color:  rgb(134, 199, 99);</string>
     </property>
     <property name="text">
      <string>별점/댓글 일치율</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_12">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>381</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>워드 클라우드</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsView_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>381</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;color: black;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;font: 75 9pt &amp;quot;나눔고딕 ExtraBold&amp;quot;;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;

font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;

border-radius: 15px;
background-color:  rgb(134, 199, 99);</string>
     </property>
     <property name="text">
      <string>긍정 WordCloud</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_11">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>381</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;color: black;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;font: 75 9pt &amp;quot;나눔고딕 ExtraBold&amp;quot;;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;

font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;

border-radius: 15px;
background-color:  rgb(134, 199, 99);</string>
     </property>
     <property name="text">
      <string>부정 WordCloud</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>기타 자료</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsView_13">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>381</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>381</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;color: black;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;font: 75 9pt &amp;quot;나눔고딕 ExtraBold&amp;quot;;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;

font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;

border-radius: 15px;
background-color:  rgb(134, 199, 99);</string>
     </property>
     <property name="text">
      <string>MENU TOP 5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;color: black;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;font: 75 9pt &amp;quot;나눔고딕 ExtraBold&amp;quot;;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;

font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;

border-radius: 15px;
background-color:  rgb(134, 199, 99);</string>
     </property>
     <property name="text">
      <string>항목 별 별점</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Consolas&quot;;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;
color: black;
	background-color: rgb(134, 199, 99);
	border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>사이트 별점</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(232, 232, 232);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Consolas&quot;;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;
color: black;
	background-color:rgb(255, 255, 127);
	border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Total Comment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Star_Total_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
background-color: rgb(232, 232, 232);
font: 10pt &quot;Consolas&quot;;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Consolas&quot;;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;
color: black;
	background-color: rgb(134, 199, 99);
	border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>학습한 별점</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Consolas&quot;;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;
color: black;
	background-color:rgb(255, 255, 127);
	border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>긍정/부정 댓글 비율</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Store_Name">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
background-color:  rgb(232, 232, 232);


font: 10pt &quot;Consolas&quot;;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Consolas&quot;;
font: 75 9pt &quot;나눔고딕 ExtraBold&quot;;
color: black;
	background-color:rgb(255, 255, 127);
	border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>STORE NAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color:  rgb(232, 232, 232);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Consolas&quot;;
font: 75 20pt &quot;나눔고딕 ExtraBold&quot;;
color: rgb(255, 255, 127);
	background-color: rgb(199,199,199);
	border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>G r a y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Consolas&quot;;
font: 75 20pt &quot;나눔고딕 ExtraBold&quot;;
color: rgb(121, 121, 121);
	background-color: rgb(199,199,199);
	border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>C o r n 🌽</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
